entries/entry01.md

```diff
@@ -1,8 +1,8 @@
 # Entry 1
```

Comment:
Cool! Missing sources, EDP, skills. And I'm not taking off points for professionalism, but next time use more markdown syntax to create visual separation.